Update release notes for April 2026 (#491)
canonical/launchpad-manual: docs/release-notes.rst

Release Notes
=============

April 2026
++++++++++

13 April

> - Added build_path option to snap package recipes, allowing for having multiple

Reviewer: ```suggestion
- Added the `build_path` option to snap package recipes, allowing multiple
```

> snapcraft.yaml in the same repository.

Reviewer: ```suggestion
`snapcraft.yaml` in the same repository.
```
